src/lib/draw/Toolbox.svelte

```diff
@@ -59,38 +62,38 @@
       "street-view": streetViewMode,
     };
 
-    if (mode == newMode) {
+    const mode = get(currentMode);
```

Comment:
Can't we just do `const mode = $currentMode`? I think it should compile down to the same thing, and also magically insert the unsubscribe: https://svelte.dev/tutorial/auto-subscriptions